Source/Editor/Utilities/Utils.cs

```diff
@@ -1656,8 +1657,9 @@ public static void ShowSourceCodeWindow(string source, string title)
             settings.AllowMaximize = false;
             settings.AllowMinimize = false;
             settings.HasSizingFrame = false;
-            settings.StartPosition = WindowStartPosition.CenterScreen;
-            settings.Size = new Vector2(500, 600) * Platform.DpiScale;
+            settings.StartPosition = WindowStartPosition.CenterParent;
```

Comment:
When merging this, please do check out this piece of code since we might get a merge conflict here.

Reply:
ok